### PR TITLE
Fix issue with the logs directory not being present

### PR DESCRIPTION
**Pull request by arilivigni**
- This directory doesn't exist on the duffy hose
- Link to output directory

### CODE REVIEW

**Comment by arilivigni** [comment]
I know I am supposed to be on PTO but it is raining up here and we are all inside the cabins ;)

File: tasks/ostree-image-compose [hunk 1]
```diff
@@ -10,6 +10,7 @@ base_dir="$(dirname $0)/.."
 HOMEDIR=$(pwd)
 rm -rf ${HOMEDIR}/output
 mkdir -p ${HOMEDIR}/output
+ln -fs ${HOMEDIR}/output ${HOMEDIR}/logs
```
Comment (p3ck) [inline tasks/ostree-image-compose]: line 32 should mean this isn't required.

Comment (arilivigni) [inline tasks/ostree-image-compose]: weird I thought it ran with your change and failed for the same reason when I went to the node for that stage name logs was not there.

Comment (arilivigni) [inline tasks/ostree-image-compose]: Actually probably should have made this ln -fs ${HOMEDIR}/output/logs ${HOMEDIR}/logs

**Pull request closed without merging**